proposals/handler.go

```diff
@@ -346,10 +346,6 @@ func (h *Handler) checkBallotDataIntegrity(b *types.Ballot) error {
 		return errInvalidATXID
 	}
 
-	if b.Votes.Base == types.EmptyBallotID {
```

Comment:
it is syntactically valid to have no base ballot. for example if at layer genesis + 3, it is valid to use not base ballot to add support votes for blocks in layers genesis + 1 and + 2

Reply:
i see. thanks for the explanation

Reply:
i don't see why. if ballot doesn't vote for a block it can simply drop it from the support list
we still need to enforce max distance to the base ballot though

Reply:
what confuses me is, say all ballots dont vote for historical blocks and are still considered valid, how does any block cross the global threshold? isnt this an attack against the tortoise protocol?

Reply:
sure when a node dont have a block locally, it doesnt have the block in the support list. i thought thats then considered as an against vote in the full tortoise mode.

Reply:
> how does any block cross the global threshold? 

it means that all ballots votes against such block and it will cross negative threshold

> isnt this an attack against the tortoise protocol?

tortoise counts votes. i don't see an attack here

> i thought thats then considered as an against vote in the full tortoise mode.

against votes are counted by verifying mode as well

Reply:
ok. thanks. the protocol depends on 2/3 honest actors and honest miners are incentivized to vote for historical blocks.